### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by wizardwatch** [sawshep/sawsge#1]
Some documentation would be nice, as it is now it the code is fairly unapproachable.

**Comment by sawshep** [comment]
I agree. I'll probably use the [RDoc](https://ruby.github.io/rdoc/) specification.
